go.mod

```diff
@@ -1,23 +1,22 @@
 module github.com/giantswarm/release-operator
 
-go 1.13
+go 1.14
 
 require (
-	github.com/giantswarm/apiextensions v0.4.20
+	github.com/giantswarm/apiextensions/v2 v2.1.0
 	github.com/giantswarm/exporterkit v0.2.0
-	github.com/giantswarm/k8sclient/v3 v3.1.2
+	github.com/giantswarm/k8sclient/v4 v4.0.0
 	github.com/giantswarm/microendpoint v0.2.0
 	github.com/giantswarm/microerror v0.2.1
-	github.com/giantswarm/microkit v0.2.1
+	github.com/giantswarm/microkit v0.2.2-0.20200901165545-4a16870c8303
 	github.com/giantswarm/micrologger v0.3.1
-	github.com/giantswarm/operatorkit v1.2.0
+	github.com/giantswarm/operatorkit/v2 v2.0.0
 	github.com/google/go-cmp v0.5.1
 	github.com/prometheus/client_golang v1.7.1
 	github.com/spf13/viper v1.7.1
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
-	sigs.k8s.io/controller-runtime v0.5.2
+	k8s.io/apimachinery v0.18.5
+	k8s.io/client-go v0.18.5
+	sigs.k8s.io/controller-runtime v0.6.1
 )
 
-// v3.3.17 is required by sigs.k8s.io/controller-runtime v0.5.2. Can remove this replace when updated.
-replace github.com/coreos/etcd v3.3.17+incompatible => github.com/coreos/etcd v3.3.24+incompatible
+replace github.com/bketelsen/crypt => github.com/bketelsen/crypt v0.0.3
```

Comment:
See https://github.com/giantswarm/microkit/pull/147#issue-477189949 until there is a new viper release we need this replace. 

Otherwise Nancey fires for an Etcd vuln.